src/IntegerDivisionTable.cpp

```diff
@@ -9,7 +9,7 @@
  * expensive method, while the compile-time set uses
  * the cheapest method for the given divisor.
  */
-#include <stdint.h>
+#include <cstdint>
```

Comment:
The signed tables actually have 257 entries, but are declared as 256. I'm assuming that 257 entries is a mistake on the part of find_inverse.c.